mesop/server/config.py

```diff
@@ -0,0 +1,27 @@
+import os
+from typing import Literal
+
+from dotenv import load_dotenv
+from pydantic import BaseModel
+
+load_dotenv()
+
+
+class Config(BaseModel):
+  """Mesop app configuration."""
+
+  state_session_backend: Literal["memory", "none"] = "memory"
+
+  @property
+  def state_session_enabled(self):
+    return self.state_session_backend != "none"
+
+
+def CreateConfigFromEnv() -> Config:
+  return Config(
+    state_session_backend=os.getenv("MESOP_STATE_SESSION_BACKEND", "memory"),
```

Comment:
I think we should make this opt-in initially - I'm a little concerned we will hit OOMs (see other comment), and then later make this the default

Reply:
I agree. That definitely a safer roll out. Updated.